changelog-2025.09.15-18.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index d3728e1a98..59fb97f9ee 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -115,6 +115,21 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2025.09.15-18" date="2025-09-15">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>OSM data as of September 13</li>
```

Comment:
* New OpenStreetMap data as of September 13
* Removed minor islands from the World map
* Show Postcode/zip code for addresses
* Fixed wrong map centering around the current position
* Bookmark colors are preserved when exporting and importing GPX files
* Updated translations

Map Styles:
* Display lighting shops on the map
* Show power lines from zoom 18
* Show reference names for power stations and substations
* Show campsites and caravan sites in navigation mode
* Fixed secondary highway color in navigation mode
* Draw national park borders
* Draw archaeological sites from zoom 12 in outdoor mode